docs/concepts/operator.md

```diff
@@ -232,6 +232,106 @@ Please refer to [this document](../usage/shoot_credentials_rotation.md#gardener-
 
 ⚠️ Rotation of static `ServiceAccount` secrets is not supported since the `kube-controller-manager` does not enable the `serviceaccount-token` controller.
 
+## Upgrading An Existing Gardener Landscape To `gardener-operator`
```

Comment:
Should we rather call it migrating instead of upgrading? The ownership/responsibility is migrated from something else, e.g. `garden-setup` to `gardener-operator`.